modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/ShareFormModal/Email.es.js

```diff
@@ -219,23 +219,30 @@ Email.PROPS = {
 	 * @memberof Email
 	 * @type {!string}
 	 */
-	autocompleteUserURL: Config.string(),
+	autocompleteUserURL: {
+		validator: validators.string,
+	},
 
 	/**
 	 * @default undefined
 	 * @instance
 	 * @memberof Email
 	 * @type {!object}
 	 */
-	localizedName: Config.object(),
+	localizedName: {
+		validator: validators.object,
+	},
 
 	/**
 	 * @default undefined
 	 * @instance
 	 * @memberof Email
 	 * @type {!spritemap}
 	 */
-	spritemap: Config.string().required(),
+	spritemap: {
+		required: true,
+		validator: validators.string(),
```

Comment:
Looks like a bug.

Reply:
Thanks for the catch, will go over all of those now, once again when changing from `validators.string` to `validateString`